roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -23,26 +27,36 @@
     ceph_stable_rh_storage and
     not ceph_stable_rh_storage_cdn_install and
     not ceph_stable_rh_storage_iso_install
+  tags:
+    - package-install
 
 - name: make sure journal_size configured
   fail:
     msg: "journal_size must be configured. See http://ceph.com/docs/master/rados/configuration/osd-config-ref/"
-  when: journal_size|int == 0
+  when:
+    journal_size|int == 0 and
+    osd_group_name in group_names
```

Comment:
Unfortunately I'm afraid this will break the `ceph.conf` build. Assuming we leave it like and that we first deploy monitors and then later osd, the `ceph.conf` will change which will result in monitors being restarted.

Obviously, this won't affect your deployment when you only deploy monitors. However how realistic is it? :)
Same issue for `cluster_network` and `public_network`.